src/main/java/com/shippo/model/Address.java

```diff
@@ -38,13 +38,26 @@
 
     public class ValidationResults {
     	boolean is_valid;
-    	List<ValidationMessages> messages;
+    	List<ValidationMessage> messages;
+
+    	public List<ValidationMessage> getValidationMessages(){ return messages; }
+     	public void setValidationMessages(){ this.messages = messages; }
```

Comment:
Since messages cannot be posted back, only read, does it make sense to have a setter for it?